tests/orquestra/quantum/measurements/measurements_test.py

```diff
@@ -348,7 +352,11 @@ def test_get_expectation_values_from_measurements_with_bessel_correction(self):
         measurements = Measurements(
             [(0, 1, 0), (0, 1, 0), (0, 0, 0), (1, 0, 0), (1, 1, 1)]
         )
-        ising_operator = IsingOperator("10[] + [Z0 Z1] - 15[Z1 Z2]")
+        ising_operator = (
+            PauliTerm("I0", 10)
+            + PauliTerm.from_str("1*Z0*Z1")
```

Comment:
I'm wondering if we shouldn't allow for:
`PauliTerm("1*Z0*Z1")` as well?
I mean without `from_str`?
Maybe not a good idea, not sure about it 🤔